### PR TITLE
Update doc with changelog

### PR DESCRIPTION
**Pull request by xing-yang**

### CODE REVIEW

File: book/src/kubernetes-changelog.md [hunk 1]
```diff
@@ -3,6 +3,42 @@
...
+    * Azure File CSI migration
+    * vSphere CSI migration
+* Alpha
+    * Cross namespace volume provisioning
+
+## Kubernetes 1.25
+
+### Features
+* GA
+    * CSI inline volume
```
Comment (msau42) [inline book/src/kubernetes-changelog.md:20]: I think we called this CSI ephemeral volumes

File: book/src/kubernetes-changelog.md [hunk 2]
```diff
@@ -3,6 +3,42 @@
...
+* Beta
+    * vSphere CSI Migration (on by default)
+    * Portworx CSI Migration (off-by-default)
+
+## Kubernetes 1.24
+
+### Features
+* GA
+    * Volume expansion
+    * CSI storage capacity tracking
```
Comment (msau42) [inline book/src/kubernetes-changelog.md:33]: just "Storage capacity tracking" to match the previous entries

File: book/src/kubernetes-changelog.md (hunk 1 above)
Comment (xing-yang) [reply book/src/kubernetes-changelog.md]: Changed to "CSI ephemeral inline volumes":
https://kubernetes-csi.github.io/docs/ephemeral-local-volumes.html#csi-ephemeral-inline-volumes

File: book/src/kubernetes-changelog.md (hunk 2 above)
Comment (xing-yang) [reply book/src/kubernetes-changelog.md]: Done

**Pull request merged**